Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -119,6 +159,23 @@ def bump_formula_pr
 
     backup_file = File.read(formula.path) unless ARGV.dry_run?
 
+    if new_mirror
+      replacement_pairs << [/^( +)(url \"#{new_url}\"\n)/m, "\\1\\2\\1mirror \"#{new_mirror}\"\n"]
+    end
+
+    if forced_version && forced_version != "0"
+      if File.read(formula.path).include?("version \"#{old_formula_version}\"")
+        replacement_pairs << [old_formula_version.to_s, forced_version]
+      else
+        if new_mirror
+          replacement_pairs << [/^( +)(mirror \"#{new_mirror}\"\n)/m, "\\1\\2\\1version \"#{forced_version}\"\n"]
+        else
+          replacement_pairs << [/^( +)(url \"#{new_url}\"\n)/m, "\\1\\2\\1version \"#{forced_version}\"\n"]
+        end
+      end
+    elsif forced_version && forced_version == "0"
```

Comment:
How do you feel about passing an empty string `--version=""` instead of `0` to indicate an obsolete version?

Apologies for the bike-shedding; this is a very nice pull request!